components/icon/Icon.razor.cs

```diff
@@ -73,8 +73,6 @@ protected override async Task OnInitializedAsync()
                 Spin = true;
             }
 
-            await SetupSvgImg();
```

Comment:
We need to render the icons in the prerendering stage.
Therefore, the twotone colors should not be replaced during pre-rendering and should be replaced after the first rendering.

Reply:
done!